prime-dotnet-webapi/Models/SiteRegistration/IndividualDeviceProvider.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Newtonsoft.Json;
+
+namespace Prime.Models
+{
+    [Table("IndividualDeviceProvider")]
+    public class IndividualDeviceProvider : BaseAuditable
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public int CommunitySiteId { get; set; }
+
+        [JsonIgnore]
```

Comment:
`[JsonIgnore]` on our domain models was a band-aid because we didn't use View Models properly. Lets avoid adding it to new models where possible.